src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonParameterInfoOfT.cs

```diff
@@ -14,29 +12,21 @@ namespace System.Text.Json
     /// </summary>
     internal class JsonParameterInfo<T> : JsonParameterInfo
     {
-        private JsonConverter<T> _converter = null!;
-        private Type _runtimePropertyType = null!;
-
-        public override JsonConverter ConverterBase => _converter;
```

Comment:
Yes when calling the "typed" version Of ReadJson. Now that code just casts the base's converter reference to `JsonParamterInfo<T>` instead of having another reference to hold the "typed" version. Saves a bit on memory and generic variable expansion size in crossgen and doesn't appear to affect perf (the cast is very fast).